packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-glossary.jsx

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const ContractTypeGlossary = ({ category }) => {
+    let ContractTypeGlossaryTemplate;
+    if (category) {
+        switch (category) {
+            case 'accumulator':
+                ContractTypeGlossaryTemplate = (
+                    <React.Fragment>
+                        <Text as='h2' weight='bold' className='contract-type-info__content-glossary--heading'>
+                            {localize('Growth rate')}
+                        </Text>
+                        <Text as='p'>
+                            {localize('You can choose a growth rate with values of 1%, 2%, 3%, 4%, and 5%.')}
+                        </Text>
+                        <Text as='h2' weight='bold' className='contract-type-info__content-glossary--heading'>
+                            {localize('Range')}
+                        </Text>
+                        <Text as='p'>
+                            {localize(
+                                'It is a percentage of the previous spot price. The percentage rate is based on your choice of the index and the growth rate.'
+                            )}
+                        </Text>
+                        <Text as='h2' weight='bold' className='contract-type-info__content-glossary--heading'>
+                            {localize('Previous spot price')}
+                        </Text>
+                        <Text as='p'>{localize('Spot price on the previous tick.')}</Text>
+                        <Text as='h2' weight='bold' className='contract-type-info__content-glossary--heading'>
+                            {localize('Payout')}
+                        </Text>
+                        <Text as='p'>{localize('Payout is the sum of your initial stake and profit.')}</Text>
+                        <Text as='h2' weight='bold' className='contract-type-info__content-glossary--heading'>
+                            {localize('Slippage risk')}
+                        </Text>
+                        <Text as='p'>
+                            {localize(
+                                'The spot price may change by the time your order reaches our servers. When this happens, your payout maybe affected.'
+                            )}
+                        </Text>
```

Comment:
@henry-deriv all the above paragraphs are absolutely identical, and all h2s are identical among each other too. I suggest putting all content into an array and mapping through it 🙂:
```
const content = [
    { type: 'heading', text: localize('Growth rate') },
    {
        type: 'paragraph',
        text: localize('You can choose a growth rate with values of 1%, 2%, 3%, 4%, and 5%.'),
    },
    // the rest of the content
];
// in jsx:
<React.Fragment>
{content.map(({ type, text }) =>
    type === 'heading' ? (
        <Text
            as='h2'
            key={text.substring(0, 10)}
            weight='bold'
            className='contract-type-info__content-glossary--heading'
        >
            {text}
        </Text>
    ) : (
        <Text as='p' key={text.substring(0, 10)}>
            {text}
        </Text>
    )
)}
</React.Fragment>
```

Reply:
i didn't put it as a content map because of <ul><li> on line 70.

Reply:
@henry-deriv i've updated my suggestion above with example for vanillas with ul and li🙂

Reply:
makes sense